go/runtime/host/sgx/ecdsa.go

```diff
@@ -144,14 +144,15 @@ func (ec *teeStateECDSA) Update(ctx context.Context, sp *sgxProvisioner, conn pr
 	// also do their own verification).
 	// Check bundles in order: fresh first, then cached, then try downloading again if there was
 	// no scheduled refresh this time.
-	tcbBundle, err := func() (*pcs.TCBBundle, error) {
+	getTcbBundle := func(update pcs.UpdateType) (*pcs.TCBBundle, error) {
 		var fresh *pcs.TCBBundle
 
 		cached, refresh := ec.tcbCache.check(pckInfo.FMSPC)
```

Comment:
Does the cache need to distinguish between early and standard (e.g. to overwrite early when standard is available)?

Reply:
No, it doesn't matter which TCB info is used as long as it passes verification against the quote policy. Over time standard becomes the same as early anyway.